application/testing/CMakeLists.txt

```diff
@@ -1037,8 +1037,8 @@ endif()
 f3d_test(NAME TestRenderingBackendInvalid DATA cow.vtp RENDERING_BACKEND invalid ARGS --verbose REGEXP "rendering-backend value is invalid, falling back to" NO_BASELINE)
 
 # Test that f3d can try to read a system config file
-add_test(NAME f3d::TestNoDryRun COMMAND $<TARGET_FILE:f3d> --no-render)
-set_tests_properties(f3d::TestNoDryRun PROPERTIES TIMEOUT 4)
+add_test(NAME f3d::TestNoNoRender COMMAND $<TARGET_FILE:f3d> --no-render)
+set_tests_properties(f3d::TestNoNoRender PROPERTIES TIMEOUT 4)
```

Comment:
this one is incorrect. It should be `TestNoNoConfig` and use `no-config` instead of `no-render`.

Reply:
The idea is that all test uses `no-config` so we add a test that do nother but **not** use `no-config`.
It needs `no-render` because without it F3D starts in interactive mode.

Reply:
> this one is incorrect. It should be `TestNoNoConfig`. It uses `no-render` but this is expected.

Like this? dbbf44ba1863a21274a264806e8382d10e0e53d4

Reply:
yes